packages/godwoken/src/types.ts

```diff
@@ -54,3 +55,8 @@ export interface AccountMerkleState {
   merkle_root: Hash;
   count: HexU32;
 }
+
+export interface RegistryAddress {
+  registry_id: HexNumber;
```

Comment:
I think `HexU32` is more accurate than `HexNumber`.

Reply:
updated